zebra/zebra_ns.h

```diff
@@ -34,6 +34,18 @@ struct nlsock {
 };
 #endif
 
+struct zebra_ns_tables {
+	RB_ENTRY(zebra_ns_tables) zebra_ns_tables_entry;
+
+	uint32_t tableid;
+	afi_t afi;
+
+	struct route_table *table;
+};
+RB_HEAD(zebra_ns_tables_head, zebra_ns_tables);
+RB_PROTOTYPE(zebra_ns_tables_head, zebra_ns_tables, zebra_ns_tables_entry,
+	     zebra_ns_tables_entry_compare)
```

Comment:
should there be a semicolon here for consistency with the previous line?

Reply:
I was following consistency with all the other places we've done this :)